### DISCUSSION TITLE
Just found Todo mode and wanted to say thanks 🙂

### DISCUSSION THREAD

**Discussion by will-stone** [rxhanson/Rectangle#608] [General]
Thank you @rxhanson for continuing to support this app. I use it daily. I've just found Todo mode and it's really useful for keeping a terminal on the right whilst working.

**Comment by rxhanson** [comment]
Glad you’re enjoying Rectangle! Todo mode was created a while back by Patrick Collison and Nat Friedman, but I finally got around to adding it to the UI in the latest release.
